trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Lift/compute_RHS_matrix.py

```diff
@@ -0,0 +1,112 @@
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+# compute_RHS_matrix.py
+# 
+# Created:  Aug 2018, M. Clarke
+#           Apr 2020, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports
+import numpy as np
+
+## @ingroup Methods-Aerodynamics-Common-Fidelity_Zero-Lift
+def compute_RHS_matrix(VD,n_sw,n_cw,delta,phi,conditions,geometry,sur_flag):     
+
+    # unpack 
+    propulsors       = geometry.propulsors
+    aoa              = conditions.aerodynamics.angle_of_attack 
+    aoa_distribution = np.repeat(aoa, VD.n_cp, axis = 1) 
+    V_inf            = conditions.freestream.velocity
+    V_distribution   = np.repeat(V_inf , VD.n_cp, axis = 1)
+    m                = len(aoa) # number of control points      
+    
+    #-------------------------------------------------------------------------------------------------------
+    # PROPELLER SLIPSTREAM MODEL
+    #-------------------------------------------------------------------------------------------------------
+    '''
+    SOURCE: Aerodynamic Modelling of the Wing-Propeller Interaction
+    Summary: This method uses the blade element momentum solution to modify the local angle of 
+    attach and axial velocity incident on the wing
+    The code below assumes that if the propeller is interacting with the wing, the wing is always situated at the 
+    center of the propeller 
+    '''
+    if sur_flag == False:
```

Comment:
you should make these if A and B:

Reply:
Corrected this as well as one later in the script